src/tools/PrizeTierController/ProjectionView/PrizePoolItem.tsx

```diff
@@ -0,0 +1,409 @@
+                <div>{getTimeBasedDrawValue(prizeChances[i], 'day')}</div>
+                <div>{getTimeBasedDrawValue(prizeChances[i], 'week')}</div>
+                <div>{getTimeBasedDrawValue(prizeChances[i], 'month')}</div>
+                <div>{getTimeBasedDrawValue(prizeChances[i], 'year')}</div>
+              </li>
+            )
+          })}
+        </ul>
+      )}
+    </div>
+  )
+}
+
+const AdvancedOptions = (props: {
+  tvl: number
+  errors: FieldErrorsImpl<ProjectionSettings>
+  register: UseFormRegister<ProjectionSettings>
+  watch: UseFormWatch<ProjectionSettings>
+  setValue: UseFormSetValue<ProjectionSettings>
+}) => {
+  const { tvl, errors, register, watch, setValue } = props
+  const [isCollapsed] = useAtom(isListCollapsed)
+  const { t } = useTranslation()
+
+  return (
+    <div className={classNames({ hidden: isCollapsed })}>
+      <SectionTitle text={t('advancedOptions')} className='mb-2' />
+      <ProjectionInput
+        title='TVL'
+        formKey='tvl'
+        validate={{
+          isValidNumber: (v) => !Number.isNaN(Number(v)) || t('fieldIsInvalid', { field: 'TVL' }),
+          isGreaterThanOrEqualToZero: (v) =>
+            parseFloat(v) >= 0 || t('fieldIsInvalid', { field: 'TVL' })
+        }}
+        errors={errors}
+        register={register}
+        watch={watch}
+        setValue={setValue}
+        onResetValue={tvl.toFixed(0)}
+      />
+      <ProjectionInput
+        title={t('variance')}
+        formKey='variance'
+        validate={{
+          isValidNumber: (v) =>
+            !Number.isNaN(Number(v)) || t('fieldIsInvalid', { field: t('variance') }),
+          isValidPercentage: (v) =>
+            parseFloat(v) >= -100 || t('fieldIsInvalid', { field: t('variance') })
+        }}
+        errors={errors}
+        register={register}
+        watch={watch}
+        setValue={setValue}
+        onResetValue='0'
+        percent
+      />
+    </div>
+  )
+}
+
+const SectionTitle = (props: { text: string; className?: string }) => {
+  return <h3 className={classNames('text-xs opacity-80', props.className)}>{props.text}</h3>
+}
+
+const ProjectionInput = (props: {
+  title: string
+  formKey: keyof ProjectionSettings
+  validate?: Record<string, (v: any) => true | string>
+  disabled?: boolean
+  errors: FieldErrorsImpl<ProjectionSettings>
+  register: UseFormRegister<ProjectionSettings>
+  watch: UseFormWatch<ProjectionSettings>
+  setValue: UseFormSetValue<ProjectionSettings>
+  onResetValue?: string
+  className?: string
+  percent?: boolean
+}) => {
+  const {
+    title,
+    formKey,
+    validate,
+    disabled,
+    errors,
+    register,
+    watch,
+    setValue,
+    onResetValue,
+    className,
+    percent
+  } = props
+  const { t } = useTranslation()
+
+  const formValue = watch(formKey)
+
+  return (
+    <div className={classNames(className)}>
+      <Label className='uppercase' htmlFor={formKey}>
```

Comment:
nit: you'll end up with multiple html elements with the same ids. Try clicking a label on the last Prize Pool form

Reply:
Fixed through 0d1b7a0